DigitalLearningSolutions.Web/Scripts/trackingSystem/reports.ts

```diff
@@ -1,70 +1,140 @@
 import Chartist from 'chartist';
 import getPathForEndpoint from '../common';
 
+// These constants are used in _ActivityTable.cshtml
+const toggleableActivityButtonClass = 'js-toggle-row-button';
+const toggleableActivityRowClass = 'js-toggleable-activity-row';
+
+const path = getPathForEndpoint('TrackingSystem/Centre/Reports/Data');
+const activityToggleableRowDisplayNone = 'none';
+const activityToggleableRowDisplayTableRow = 'table-row';
+
 interface IActivityDataRowModel {
-  period: string;
-  completions: number;
-  evaluations: number;
-  registrations: number;
+	period: string;

```

Comment:
Is this from auto-formatting? Have a look in your IDE code style settings to make sure you've got it set up right.
We should:
- use spaces instead of tabs 
- Have space on each side of a brace